xbmc/video/dialogs/GUIDialogVideoSettings.cpp

```diff
@@ -259,8 +259,6 @@ void CGUIDialogVideoSettings::InitializeSettings()
   entries.push_back(std::make_pair(16318, VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL_HALF));
   entries.push_back(std::make_pair(16317, VS_INTERLACEMETHOD_VDPAU_TEMPORAL_HALF));
   entries.push_back(std::make_pair(16314, VS_INTERLACEMETHOD_VDPAU_INVERSE_TELECINE));
-  entries.push_back(std::make_pair(16320, VS_INTERLACEMETHOD_DXVA_BOB));
```

Comment:
we had this all the time with this spinner: suppose you have selected Software rendering and choose "Deinterlace". Then save the setting. Go back to acceleration and say "dxva rendering". Now the options in the spinner are all gone, and the selection of "Deinterlace" (saved in the db) doesn't cause issues. Instead, it goes back to "Automatic". I say they are robust enough.